test/module/Entra/Remove-EntraUser.Tests.ps1

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+
+BeforeAll {  
+    if ((Get-Module -Name Microsoft.Graph.Entra) -eq $null) {
+        Import-Module Microsoft.Graph.Entra       
+    }
+    Import-Module (Join-Path $psscriptroot "..\Common-Functions.ps1") -Force
+
+
+    Mock -CommandName Remove-MgUser -MockWith {} -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "Remove-EntraUser" {
+    Context "Test for Remove-EntraUser" {
+        It "Should return empty object" {
+            $result = Remove-EntraUser -ObjectId "199a9eb1-2de2-41f2-91a6-d6444e59afb2"
```

Comment:
Replace ObjectId with dummy IDs in all places in this PR

Reply:
Updated!